client/my-sites/plugins/main.jsx

```diff
@@ -220,6 +221,10 @@ export default React.createClass( {
 		}
 	},
 
+	goToBrowser() {
```

Comment:
The name here doesn't quite match what happens on the function? I was expecting to have redirect logic here.